### ISSUE TITLE
Accessibility for the Post Widget, add aria-label or title to the Read More links and Thumbnail link

### ISSUE THREAD

**Issue by dacxjo** [elementor/elementor#13784]
## Prerequisites

- [ X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [ X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
This will give to the the thumbnail image link and the read more link from the post a discernible name, and help to pass the Google lightouse or pagespeed  "Links do not have a discernible name" test

**Describe the solution you'd like**
Add this attribute to the links by default

**Describe alternatives you've considered**
Define on which html element to set the Elementor Custom attributes 
Have an section to set the accessibility related config

**Additional context**
Using the custom attributes for the post element link will only set the attribute to the wrapper instead of the link.

**Comment by Bovelett** [comment]
+1 :-)

**Comment by JboyJW** [comment]
+1

**Comment by lauradotcom** [comment]
+1! This issue impacts both accessibility and SEO. Even coding a custom function to solve the issue requires advanced plugin developer skills because the togglable "read more" link is not targetable via standard WP functions/hooks. Please make accessibility a higher priority!

**Comment by nicholaszein** [comment]
Hello, everyone!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro **v3.11**! 🥳 

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂
